CHANGELOG.md

```diff
@@ -2,9 +2,9 @@
 
 ### 1.3.2
 
-- Abandoned cart sent status is reset when customer empties cart [#73]
-- Abandoned cart status is now calculated from last item adding time instead of the first item [#75]
-- Abandoned cart activation time is used to calculate the oldest abandoned cart. Abandoned cart emails are no longer sent to very old carts. [#72]
+- Abandoned cart sent status is reset when customer empties cart [#73](https://github.com/sendsmaily/smaily-opencart-module/issues/73)
```

Comment:
Currently the tag is just present without square brackets around it.
Should I add those:
Abandoned cart sent status is reset when customer empties cart [[#73](https://github.com/sendsmaily/smaily-opencart-module/issues/73)]

Reply:
Yeah, square brackets would be nice. At the moment in kind of blends in with the issue description.

Reply:
Added square brackets 
0a98b8d